### PR TITLE
Add content to home page and enhance style

### LINKED ISSUE #53: Give home page bullets real content
Give 'bullet' items on the home page real content.

- sanborn maps
- data types
- ??

### PR DESCRIPTION
**Pull request by eliotjordan**
Adds initial content to home page. Cleans up some styling issues. Closes #53 #54 and #111.

### CODE REVIEW

File: app/assets/javascripts/modules/locationsearch.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ $(document).ready(function() {
       $('.leaflet-container').toggleClass('active');
     });
 
+    // hide zoom control on load
+    $('.leaflet-control-zoom').toggle();
```
Comment (tampakis) [inline app/assets/javascripts/modules/locationsearch.js]: Should the zoom control reappear if the user toggles on the map search from the home page?

File: app/views/catalog/_home.html.erb [hunk 2]
```diff
@@ -1,29 +1,88 @@
...
+      </a>
+      <a href="/?f%5Bdc_subject_sm%5D%5B%5D=population" class="grid-item">
+        <span class="icon-graphbar"></span>
+        <h1>Census</h1>
+        <p>Worldwide population and demographic data.</p>
+      </a>
+        <a href="/?f%5Bdct_spatial_sm%5D%5B%5D=Princeton&q=" class="grid-item">
+        <span class="geoblacklight-icon geoblacklight-princeton"></span>
+        <h1>Princeton</h1>
+        <p>Data and maps related Princeton University.</p>
```
Comment (tampakis) [inline app/views/catalog/_home.html.erb]: Data and maps related to Princeton University

Comment (eliotjordan) [inline app/views/catalog/_home.html.erb]: Derp. Yes. I did graduate from college. I swear!

File: app/views/catalog/_home.html.erb [hunk 3]
```diff
@@ -1,29 +1,88 @@
...
+    <div class="line-behind-text">
+      <h6 class="er_toc_tag" id="er-toc-id-2">Featured Maps</h6>
     </div>
-    <div class="bullet-content">
-      <h2>Sanborn</h2>
-      <p>Detailed building maps of cities and towns in New Jersey. Created by The Sanborn Map Company, the oldest mapping company in the United States, primarily for use by insurance companies.</p>
+
+    <div class="cards">
+      <div class="card">
+        <a class="nostyle" href="/catalog/princeton-2v23vw81v">
```
Comment (tampakis) [inline app/views/catalog/_home.html.erb]: The featured maps look cool.

File: app/assets/javascripts/modules/locationsearch.js (hunk 1 above)
Comment (eliotjordan) [inline app/assets/javascripts/modules/locationsearch.js]: I'm trying out a minimal look. Not sure how useful those zoom buttons are in this context. Plus there's some weird CSS issues with the buttons after you do a location search that I haven't been able to track down.

**Comment by eliotjordan** [comment]
Just force pushed an update.

File: app/views/catalog/_home.html.erb (hunk 3 above)
Comment (eliotjordan) [inline app/views/catalog/_home.html.erb]: Yes, those came out nice. I was thinking that we could have Axa look over the app and make some notes about how she would improve the ui. I did my best.

**Pull request merged**